src/core/adapters/recommendations.ts

```diff
@@ -64,6 +65,42 @@ namespace Recommendations {
       transform({ results, navigations, config }), availableNavigations);
   };
 
+  export const addLocationToRequest = (request, state, config: Configuration):
+  RecommendationsBody | RecommendationsRequest => {
```

Comment:
even if it causes the line to go long, I prefer to keep types always to the right of the associated token(s) for clarity